### ISSUE TITLE
Accessibility fixes

### ISSUE THREAD

**Issue by JsephGriff** [slweb-uw/volunteer-site#151]
I have created a branch called accessibility-fix which currently contains fixes for our website's ```title``` HTML.

Below are listed the remaining changes that need to be made, and their relevant associated pages.

- Missing alt text/ decorative images not hidden URLS of Affected Pages:
[https://servicelearning.washington.edu/requiredTraining](https://uw.dubbot.com/a/61ae2795a7d37f0001354842/sites/680fda3c8ea89967b313b831/pages/680fda6d7bac9fa409d9c857?tab=accessibility&issueId=image-alt), [https://servicelearning.washington.edu/welcome/volunteerProviders](https://uw.dubbot.com/a/61ae2795a7d37f0001354842/sites/680fda3c8ea89967b313b831/pages/680fda737bac9fa409d9c85b?tab=accessibility&issueId=image-alt), [https://servicelearning.washington.edu/welcome/communitypartners](https://uw.dubbot.com/a/61ae2795a7d37f0001354842/sites/680fda3c8ea89967b313b831/pages/680fda797bac9fa409d9c85f?tab=accessibility&issueId=image-alt), [https://servicelearning.washington.edu/standardOfCare](https://uw.dubbot.com/a/61ae2795a7d37f0001354842/sites/680fda3c8ea89967b313b831/pages/680fda807bac9fa409d9c862?tab=accessibility&issueId=image-alt), [https://servicelearning.washington.edu/welcome/categoriesOfService](https://uw.dubbot.com/a/61ae2795a7d37f0001354842/sites/680fda3c8ea89967b313b831/pages/680fda967bac9fa409d9c86e?tab=accessibility&issueId=image-alt). Decorative bar is not hidden from screen readers on multiple pages. This can be fixed by assigning the image an empty alt attribute (alt=””), or role=”none”.
- List items not contained in a list element. https://servicelearning.washington.edu/requiredTraining
- Input field missing accessible label. https://servicelearning.washington.edu/opportunities Note: Dropdown does not have an accessible name that tells screen readers what the purpose of the input field is. This can be solved by labeling the input using aria-label or aria-labelledby attributes.

**Comment by M4X1M4S** [comment]
I want to work on this.

**Comment by eemmalu** [comment]
Not super sure how to fix the input field accessibility label—as far as I can tell it does have a properly formatted field for `aria-labelledby`. I just added an additional `aria-label` field for more directly labelling it and am hoping that works.
